### PR TITLE
[DONE][Just Merge Me][Community Translation] Russian Docs Translation

### PR DESCRIPTION
**Pull request by gbezyuk**
## Translation progress

- [x] **Introduction**

#### Getting Started
- [x] Vue Component Spec
- [x] Setting Up a Project

#### Features
- [x] ES2015
- [x] Scoped CSS
- [x] CSS Modules
- [x] PostCSS
- [x] Hot Reload

#### Configurations
- [x] Pre-Processors
- [x] Asset URL Handling
- [x] Advanced Loader Configuration
- [x] Extracting CSS File

#### Workflow
- [x] Production Build
- [x] Linting
- [x] Testing
- [x] Testing with Mocks

- [x] **Options Reference**

### CODE REVIEW

**Comment by gbezyuk** [comment]
ping @kazupon — the translation is done, maybe it's time to merge it ;)

**Comment by simplesmiler** [comment]
Sorry for the delay, will give a look this evening.

File: docs/ru/options.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    }
+  }
+  ```
+
+### cssSourceMap
+
+- Тип: `Boolean`
+- По умолчанию: `true`
+
+  Whether to enable source maps for CSS. Disabling this can avoid some relative path related bugs in `css-loader` and make the build a bit faster.
```
Comment (simplesmiler) [inline docs/ru/options.md]: Непереведенный кусок (отсюда и до конца файла).

Comment (gbezyuk) [inline docs/ru/options.md]: ping @Alex-Sokolov

**Comment by gbezyuk** [comment]
ping @simplesmiler

**Comment by simplesmiler** [comment]
Changes look good :+1: 
/ping @kazupon

**Comment by gbezyuk** [comment]
@simplesmiler @kazupon please let us know when these changes will be deployed

**Comment by kazupon** [comment]
@gbezyuk 
deploy done!

**Comment by gbezyuk** [comment]
awesome! thank you!

**Pull request merged**